crypto/dh/build.info

```diff
@@ -3,7 +3,10 @@ LIBS=../../libcrypto
 $COMMON=dh_lib.c dh_key.c dh_group_params.c dh_check.c dh_backend.c
 
 SOURCE[../../libcrypto]=$COMMON\
-        dh_asn1.c dh_gen.c dh_err.c dh_depr.c \
+        dh_asn1.c dh_gen.c dh_err.c \
         dh_ameth.c dh_pmeth.c dh_prn.c dh_rfc5114.c dh_kdf.c dh_meth.c
+IF[{- !$disabled{'deprecated-3.0'} -}]
```

Comment:
This should be deprecated-0.9.8 instead.

Reply:
fixed.